packages/filebrowser/src/crumbs.ts

```diff
@@ -309,6 +319,15 @@ export class BreadCrumbs extends Widget {
   private _hasPreferred: boolean;
   private _crumbs: ReadonlyArray<HTMLElement>;
   private _crumbSeps: ReadonlyArray<HTMLElement>;
+  private _fullPath: boolean;
+
+  get fullPath(): boolean {
+    return this._fullPath;
+  }
+
+  set fullPath(value: boolean) {
```

Comment:
I'm wondering here if I should check `options.fullPath`? otherwise I think the default value in `fileBrowserConfig` might always override a true `options.fullPath`? assuming, if we go with adding the toggle command, that wouldn't check the options setting?

Reply:
This is fine as-is, the `options` is mostly for testing, ensuring there is a value to start with testing, and to enable standalone usage (when another application building upon jupyterlab core packages rewrites their own `-extension` packages)